back/db/schema.rb

```diff
@@ -86,18 +86,38 @@
     t.index ["analysis_id"], name: "index_analysis_background_tasks_on_analysis_id"
   end
 
-  create_table "analysis_summaries", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+  create_table "analysis_insights", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.uuid "analysis_id", null: false
+    t.string "insightable_type", null: false
+    t.uuid "insightable_id", null: false
+    t.jsonb "filters", default: {}, null: false
+    t.jsonb "inputs_ids"
+    t.datetime "created_at", null: false
```

Comment:
It's also possible to write `t.timestamps`.

However, I just realize that it seems that Rails does not index the timestamps by default. In the controller, the insights are ordered by `created_at`, so it seems like it could be useful in this case (depending on the number of insights we expect to have).

If this is true, then I think we might be able to make significant performance improvements on endpoints such as `ideas#index`. Do you think this could be worth investigating?

Reply:
In the general case, yes, definitely worth investigating.

In this case, we're talking about a list with items is the order of 10-100, but definitely not more, in which case a full table scan is often even faster.